### PR TITLE
Swipe to refresh is too sensitive

### PR DESCRIPTION
**Pull request by marcosholgado**
Task/Issue URL: https://app.asana.com/0/1125189844152671/1199888021167463
Tech Design URL: 
CC: 

**Description**:
This PR changes how we enable swipe to refresh so it only happens once a website is clamped at the top AND a new gesture begins. This should fix https://github.com/duckduckgo/Android/issues/1081

**Steps to test this PR**:
1. Go to any site with scrollable content, i.e. `https://m.xkcd.com/2425/`
1. When the page loads for the first time, pulling to refresh should be enabled.
1. Scroll down a bit, then up to reach the top and then back again all within the same gesture.
1. Pull to refresh should not be activated.
1. Pull to refresh is only enabled if the site is clamped at the top and you being a new gesture.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

File: app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt [hunk 1]
```diff
@@ -1396,6 +1399,8 @@ class BrowserTabFragment :
         private const val TRACKERS_INI_DELAY = 500L
         private const val TRACKERS_SECONDARY_DELAY = 200L
 
+        private const val DEFAULT_CIRCLE_TARGET_TIMES_1_5 = 96
```
Comment (malmstein) [inline app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt:1402]: isn't this default target zone `48dp` times 2?

File: app/src/main/java/com/duckduckgo/app/browser/DuckDuckGoWebView.kt [hunk 2]
```diff
@@ -98,15 +103,16 @@ class DuckDuckGoWebView : WebView, NestedScrollingChild {
                     lastY -= scrollOffset[1]
                 }
 
-                if (scrollY == 0 && lastClampedTopY && nestedOffsetY == 0) {
-                    // we have reached the top, are clamped vertically and nestedScrollY is done too -> enable swipeRefresh (by default always disabled)
+                if (canSwipeToRefresh && scrollY == 0 && lastClampedTopY && nestedOffsetY == 0) {
```
Comment (malmstein) [inline app/src/main/java/com/duckduckgo/app/browser/DuckDuckGoWebView.kt:106]: 👍

**Review by malmstein — APPROVED** [review]
just a comment regarding naming, this is really nice @marcosholgado !

File: app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt (hunk 1 above)
Comment (marcosholgado) [reply app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt:1402]: It actually is `DEFAULT_CIRCLE_TARGET` times 1.5 (64 * 1.5). I checked the code `setDistanceToTriggerSync` sets `mTotalDragDistance`. By default it does:

```
mSpinnerOffsetEnd = (int) (DEFAULT_CIRCLE_TARGET * metrics.density);
mTotalDragDistance = mSpinnerOffsetEnd;
```

**Pull request merged**